packages/smoldot-test-utils/.gitignore

```diff
@@ -0,0 +1,11 @@
+/dist
```

Comment:
I think directories should not start with /

Reply:
You're right. I copied this from another package!